src/safe_apps/views.py

```diff
@@ -7,6 +9,10 @@
 class SafeAppsListView(ListAPIView):
     serializer_class = SafeAppsResponseSerializer
 
+    @method_decorator(cache_page(60 * 60 * 1))  # Cache 1 hour
```

Comment:
For now it's set for 1h. But up for some suggestions. Can also be extracted to an environment variable if we see fit.